WordPress/jetpack_metadata/PlayStoreStrings.po

```diff
@@ -37,19 +46,9 @@ msgid ""
 "- Keyboard won’t cover up login errors\n"
 msgstr ""
 
-msgctxt "release_note_193"
-msgid ""
-"19.3:\n"
-"- Sign up through WordPress.com and create sites in Jetpack\n"
-"- Use the Reader in Jetpack\n"
-"- See notifications in new comment editor\n"
-"- Get a fix for the flickering Page/Site preview\n"
-"- Keep editor story blocks from closing when memory is low\n"
-msgstr ""
-
 #. translators: Short description of the Jetpack app to be displayed in the Play Store. Limit to 80 characters including spaces and commas!
 msgctxt "short-description"
-msgid "Get powerful security and performance tools in your pocket."
+msgid "Powerful WordPress security and performance tools."
```

Comment:
See pxLjZ-6Vi-p2#comment-18896

Note: the corresponding `jetpack_metadata/short_description.txt` file was updated in 26548d831405f663ba07187a466640d2cba8adf1 which already got merged to trunk in #16139